mods/ts/weapons/superweapons.yaml

```diff
@@ -56,19 +56,19 @@ SuicideBomb:
 	Warhead@2Res: DestroyResource
 
 IonCannon:
-	ValidTargets: Ground, Air
+	ValidTargets: Ground, Water, Air
```

Comment:
I'm able to target water terrain on normal bleed (5ceee1d6f), because `IonCannonPower` ignores the weapon target types.  The spread damage warhead hits hover/amphib actors because they keep their Ground target type while on water.